amoro-common/src/main/java/org/apache/amoro/table/TableMetaStore.java

```diff
@@ -407,13 +407,15 @@ public UserGroupInformation getUGI() {
       return ugi;
     }
 
-    private void constructKerberosUgi() throws IOException, KrbException {
+    private void constructKerberosUgi() throws Exception {
       Path confPath = generateKrbConfPath();
       String krbConfFile = saveConfInPath(confPath, KRB_CONF_FILE_NAME, krbConf);
       String keyTabFile = saveConfInPath(confPath, KEY_TAB_FILE_NAME, krbKeyTab);
       System.clearProperty(HADOOP_USER_PROPERTY);
       System.setProperty(KRB5_CONF_PROPERTY, krbConfFile);
-      sun.security.krb5.Config.refresh();
+      Class<?> classRef = Class.forName("sun.security.krb5.Config");
```

Comment:
Maybe we can create a static variable to store this method reference rather than construct it every time.
Besides, it is not required to have `sun.security.krb5.Config` in classpath, so if it is not existed, we can skip the refresh calling.

Reply:
Besides, we may add support for IBM JDK version like :https://github.com/zhengchenyu/incubator-uniffle/blob/2704aa33aa514441763e9efe284062f02aed2d25/common/src/test/java/org/apache/uniffle/common/KerberizedHadoop.java#L231